main.py

```diff
@@ -254,28 +255,29 @@ def main(cls, log_name):
 
 class GNNTrainer(Trainer):
     @staticmethod
-    def get_policy(envs, params):
+    def get_policy(envs, params, device):
         base_kwargs = params["base"]
         policy = GNNPolicy(
             envs.get_attr("orig_obs_space")[0],
             envs.get_attr("action_space")[0],
             envs.get_attr("num_actions")[0],
             base_kwargs=base_kwargs,
+            device=device,
         )
         
-        return policy, base_kwargs
+        return policy
     
     @staticmethod
     def get_env(params, device):
         env_kwargs = params["env"]
         envs = PLEnv.make_vec_envs(
-            args.seed,
-            args.num_processes,
+            params["seed"],
+            params["num_processes"],
             device,
             **env_kwargs
         )
```

Comment:
A more idiomatic way to do this in python is to use globbing (`**`) to unpack params and extract desired keys (`env`, `seed`, and `num_processes`). In general, try to pass dictionaries as kwargs to a method and avoid indexing into dictionaries with strings if possible.